app/assets/javascripts/components/user/user_leadership.es6.jsx

```diff
@@ -0,0 +1,59 @@
+class UserLeadership extends Component {
+
+  // --------------------------------------------------
+  // Props
+  // --------------------------------------------------
+  static get propTypes() {
+    return {
+      user: React.PropTypes.object.isRequired,
+    };
+  }
+
+  // --------------------------------------------------
+  // Render
+  // --------------------------------------------------
+  renderGroup() {
+    var leadership = this.props.user.leadership;
+    if (leadership) {
+      if (leadership.group) {
```

Comment:
Getting an error when I take the conditional out
`Uncaught TypeError: Cannot read property 'full_name' of undefined`